.github/workflows/ci.yml

```diff
@@ -21,9 +21,9 @@ on:  # yamllint disable-line rule:truthy
   schedule:
     - cron: '28 0 * * *'
   push:
-    branches: ['master', 'v[0-9]+-[0-9]+-test']
+    branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
   pull_request:
-    branches: ['master', 'v[0-9]+-[0-9]+-test', 'v[0-9]+-[0-9]+-stable']
+    branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
```

Comment:
This was a merge conflict and should be reverted :)

Reply:
I had no clue :) surprised to see the change here